test/lib/util.test.js

```diff
@@ -1,11 +1,36 @@
 'use strict';
 
 const expect = require('chai').expect;
+const sinon = require('sinon');
 const fs = require('fs');
 const cheerio = require('cheerio');
 
-const util = require('../../lib/util');
+const rewire = require('rewire');
+const util = rewire('../../lib/util');
 const libingester = require('../../lib/index');
+const { jsonDateParser } = require('json-date-parser');
+
+const FAKE_NOW = 1523065866010;  // 2018-04-07T01:51:06.010Z
+
+const _RSS_JSON = fs.readFileSync(__dirname + '/test_files/rss-feeds.json',
+                                  jsonDateParser);
```

Comment:
Does passing the parser do anything here? I thought it only did something when passed to `JSON.parse()`